test/new-e2e/tests/containers/dump_cluster_state.go

```diff
@@ -166,11 +166,18 @@ func dumpKindClusterState(ctx context.Context, name string) (ret string) {
 		auth = append(auth, ssh.PublicKeys(signer))
 	}
 
-	sshClient, err := ssh.Dial("tcp", *instanceIP+":22", &ssh.ClientConfig{
-		User:            "ec2-user",
-		Auth:            auth,
-		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
-	})
+	var sshClient *ssh.Client
+	err = nil
+	for _, user := range []string{"ec2-user", "ubuntu"} {
```

Comment:
Can we expose the info from test-infra ? https://github.com/DataDog/test-infra-definitions/blob/26768c9af96eb87f28a51a7f3efdc9e7c86e656a/scenarios/aws/ec2/os_resolver.go#L34

Reply:
So far, this variable is private.

The correct way to get the proper user to use for ssh would be to leverage the pulumi stack output.
But if the pulumi stack output was working, we would use directly the provided `kubeconfig` without having to do any ssh at all.

So, in the end, this whole function is a hacky workaround for the fact that pulumi stack output is empty when the pulumi stack was only partially instanciated.
As a hacky workaround, I’d rather keep it confined here and I’d rather not make `test-infra-definitions` expose things that shouldn’t be.

If we make this map public in `test-infra-definitions` today for this workaround, when the pulumi stack output will eventually be fixed, this whole function will be removed and it will be a nightmare to make this map private again because other people might have started to rely on it for no good reason.